oat-sa/tao-community
# Hotfix/INV-707/Broken tooltip

### Related to: https://oat-sa.atlassian.net/browse/INV-707

### Summary

- [extension-tao-itemqti v29.14.4.10](https://github.com/oat-sa/extension-tao-itemqti/releases/tag/v29.14.4.10) Fix a runtime error preventing from editing a tooltip in the item authoring.

![image](https://github.com/oat-sa/extension-tao-itemqti/assets/1500098/45c9fdf8-c315-4c4e-9530-ba7bc82f3c11)

- [extension-pcisample v3.6.5](https://github.com/oat-sa/extension-pcisample/releases/tag/v3.6.5) Restore the tooltip helper in the textReader PCI.

![image](https://github.com/oat-sa/extension-pcisample/assets/1500098/97fd53c9-3584-498f-b437-e7b925f3ccad)

### Details

- [extension-tao-itemqti v29.14.4.10](https://github.com/oat-sa/extension-tao-itemqti/releases/tag/v29.14.4.10)  Backport of https://github.com/oat-sa/extension-tao-itemqti/pull/2302
- [extension-pcisample v3.6.5](https://github.com/oat-sa/extension-pcisample/releases/tag/v3.6.5) When migrating the PCI to the IMS standard, we made some surgery to remove or adapt incompatible code. However, it was a tad too aggressive, and [the helper responsible for preparing the tooltip was removed together with the one that brought MathJAX](https://github.com/oat-sa/extension-pcisample/pull/87/files#diff-82c38155dd7370921087076be131fa5e089961d18643a4bdb36253e7ad5d3470L186).

### How to test
- install TAO from tao-community v2022.11.18
- check out the branch
- install from composer
- update TAO
- open an item and try to add or edit a tooltip
- have an item with the textReader PCI and some tooltips (you can use the ones attached to the ticket)
- publish a delivery (or pick up an existing one if any)
- take the test and check the tooltip works
